### PR TITLE
[MRG+1] FIX kit2fiff GIU for Ubuntu

### PR DESCRIPTION
**Pull request by christianbrodbeck**
Turns out Ubuntu has the same problem with wildcards as Windows (cf. https://github.com/mne-tools/mne-python/pull/4013)

CC @ellenlau

### CODE REVIEW

File: mne/gui/_kit2fiff_gui.py [hunk 1]
```diff
@@ -46,8 +47,8 @@
     hsp_wildcard = ['Head Shape Points (*.hsp;*.txt)|*.hsp;*.txt']
     elp_wildcard = ['Head Shape Fiducials (*.elp;*.txt)|*.elp;*.txt']
     kit_con_wildcard = ['Continuous KIT Files (*.sqd;*.con)|*.sqd;*.con']
-elif os.name == 'nt':
-    # on Windows, multiple wildcards does not seem to work
+elif sys.platform in ('win32',  'linux2'):
```
Comment (agramfort) [inline mne/gui/_kit2fiff_gui.py]: are you sure it's not a pb of lib versions?

Comment (christianbrodbeck) [inline mne/gui/_kit2fiff_gui.py]: I don't think so, on Ubuntu and macOS I get the qt4 backend, and below are the two envs, I can't spot a meaningful difference (traits, traitsui, qt, pyqt, pyface)...

Ubuntu:
```
# packages in environment at /home/christian/anaconda2/envs/mne:
#
apptools                  4.4.0                    py27_0  
cairo                     1.14.8                        0  
configobj                 5.0.6                    py27_0  
cycler                    0.10.0                   py27_0  
envisage                  4.5.1                    py27_0  
fontconfig                2.12.1                        3  
freetype                  2.5.5                         2  
glib                      2.50.2                        1  
h5py                      2.6.0               np111py27_2  
harfbuzz                  0.9.39                        2  
hdf5                      1.8.17                        1  
imageio                   2.1.2                    py27_0    conda-forge
jbig                      2.1                           0  
jpeg                      9b                            0  
libffi                    3.2.1                         1  
libgfortran               3.0.0                         1  
libiconv                  1.14                          0  
libpng                    1.6.27                        0  
libtiff                   4.0.6                         3  
libxml2                   2.9.4                         0  
matplotlib                1.5.1               np111py27_0  
mayavi                    4.5.0                    py27_0  
mkl                       2017.0.1                      0  
nibabel                   2.1.0                    py27_0    conda-forge
numpy                     1.11.3                   py27_0  
olefile                   0.44                     py27_0  
openssl                   1.0.2k                        1  
pango                     1.40.3                        1  
pcre                      8.39                          1  
pillow                    4.0.0                    py27_1  
pip                       9.0.1                    py27_1  
pixman                    0.34.0                        0  
pycairo                   1.10.0                   py27_0  
pydicom                   0.9.9                    py27_0    conda-forge
pyface                    5.1.0                    py27_0  
pygments                  2.2.0                    py27_0  
pyparsing                 2.1.4                    py27_0  
pyqt                      4.11.4                   py27_4  
pysurfer                  0.7                      py27_0    conda-forge
python                    2.7.13                        0  
python-dateutil           2.6.0                    py27_0  
pytz                      2016.10                  py27_0  
qt                        4.8.7                         4  
readline                  6.2                           2  
scikit-learn              0.18.1              np111py27_1  
scipy                     0.18.1              np111py27_1  
setuptools                27.2.0                   py27_0  
sip                       4.18                     py27_0  
six                       1.10.0                   py27_0  
sqlite                    3.13.0                        0  
tk                        8.5.18                        0  
traits                    4.6.0                    py27_0  
traitsui                  5.1.0                    py27_0  
vtk                       6.3.0                    py27_1  
wheel                     0.29.0                   py27_0  
xz                        5.2.2                         1  
zlib                      1.2.8                         3  
```

macOS:
```
# packages in environment at /Users/christian/anaconda/envs/mne:
#
apptools                  4.4.0                    py27_0  
configobj                 5.0.6                    py27_0  
cycler                    0.10.0                   py27_0  
envisage                  4.5.1                    py27_0  
freetype                  2.5.5                         2  
h5py                      2.6.0               np111py27_2  
hdf5                      1.8.17                        1  
imageio                   2.1.2                    py27_0    conda-forge
jbig                      2.1                           0  
jpeg                      9b                            0  
libpng                    1.6.27                        0  
libtiff                   4.0.6                         3  
matplotlib                1.5.1               np111py27_0  
mayavi                    4.5.0                    py27_0  
mkl                       2017.0.1                      0  
nibabel                   2.1.0                    py27_0    conda-forge
numpy                     1.11.3                   py27_0  
olefile                   0.44                     py27_0  
openssl                   1.0.2k                        1  
pillow                    4.0.0                    py27_1  
pip                       9.0.1                    py27_1  
pydicom                   0.9.9                    py27_0    conda-forge
pyface                    5.1.0                    py27_0  
pygments                  2.2.0                    py27_0  
pyparsing                 2.1.4                    py27_0  
pyqt                      4.11.4                   py27_4  
pysurfer                  0.7                      py27_0    conda-forge
python                    2.7.13                        0  
python-dateutil           2.6.0                    py27_0  
python.app                1.2                      py27_4  
pytz                      2016.10                  py27_0  
qt                        4.8.7                         4  
readline                  6.2                           2  
scikit-learn              0.18.1              np111py27_1  
scipy                     0.19.0              np111py27_0  
setuptools                27.2.0                   py27_0  
sip                       4.18                     py27_0  
six                       1.10.0                   py27_0  
sqlite                    3.13.0                        0  
tk                        8.5.18                        0  
traits                    4.6.0                    py27_0  
traitsui                  5.1.0                    py27_0  
vtk                       6.3.0                    py27_1  
wheel                     0.29.0                   py27_0  
xz                        5.2.2                         1  
zlib                      1.2.8                         3  
```

File: mne/commands/mne_kit2fiff.py [hunk 2]
```diff
@@ -40,9 +40,15 @@ def run():
...
                       help='Name of the resulting fiff file',
                       metavar='filename')
+    parser.add_option('--debug', dest='debug', action='store_true',
+                      default=False,
+                      help='Set logging level for terminal output to debug')
 
     options, args = parser.parse_args()
 
+    if options.debug:
+        mne.set_log_level('debug')
```
Comment (agramfort) [inline mne/commands/mne_kit2fiff.py]: this will make a permanent change in the config file. You should revert the change after running

Comment (christianbrodbeck) [inline mne/commands/mne_kit2fiff.py]: Does it do that? From what I can tell it just calls `logger.setLevel()` (I also called `$ mne kit2fiff` repeatedly and the log level did not seem to get stuck): https://github.com/mne-tools/mne-python/blob/master/mne/utils.py#L1239

Comment (agramfort) [inline mne/commands/mne_kit2fiff.py]: indeed it's not set_config my bad

**Comment by agramfort** [comment]
can you see why travis complains?

**Comment by christianbrodbeck** [comment]
Dataset download on Mac... I restarted it.

File: mne/gui/_kit2fiff_gui.py (hunk 1 above)
Comment (Eric89GXL) [inline mne/gui/_kit2fiff_gui.py]: Why don't we just always do it the slightly longer but always-working way? It's less branching

Comment (christianbrodbeck) [inline mne/gui/_kit2fiff_gui.py]: It also has an effect on the UI with a difference in convenience – on macOS you can select one multi-wildcard setting and you see *all* matching files. On Windows and Ubuntu you can only see one file type at a time, and you have to select different wildcards in the file browser to see them.

**Comment by agramfort** [comment]
ok +1 for MRG then
